getting-started/sandbox-quick-start-guide.md

```diff
@@ -0,0 +1,439 @@
+|Name            |Type                              
+|------------    |---------                         
+|projectId       |String                            
+|transactionId   |String                            
+|filesCount      |Integer                           
+|downloadUrls    |JSON array of downloadUrls schema 
+
+###### 4.2.3.2 downloadUrls JSON schema
+|Name                 |Type          
+|------------         |---------    
+|fileName             |String      
+|fileStatus           |String      
+|downloadUrl\*        |String       
+|validFor\*           |String      
+
+**\* Only available when file is copied to clean bucket**
+
+###### 4.2.3.3 Sample Response Payload
+|Code             |Response
+|-----------------|-----------------------------
+|200              | "projectId": "gt-enp",
+    "transactionId": "2a94862a-2a5e-4e85-bf38-98a587c08da1",
+    "filesCount": 1,
+    "downloadUrls": [
+        {
+            "fileName": "abcd.txt",
+            "fileStatus": "FileCopiedToCleanBucket",
+            "downloadUrl": "https://gt-enp-clean-sandbox-internet.s3-sandbox.gdscft.govtechstack.sg/2a94862a-2a5e-4e85-bf38-98a587c08da1/abcd.txt?AWSAccessKeyId=ASIAUDIC7Y3DCGCP26R7&Expires=1629276790&Signature=zC7ncFeVSZ4iFHOuF5K%2FQhSpc%2Bw%3D&x-amz-security-token=IQoJb3JpZ2luX2VjEGEaDmFwLXNvdXRoZWFzdC0xIkcwRQIhAJGXKuck2d343bnUF8MyGS0jPNywYSCzh8sK%2B8bSimq3AiAe6fyFwJ22ewxKJSmR4xbdSN1nRUL0wfrvlA8%2BUVs%2FFCq2AgiK%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F8BEAEaDDI4MTg2MzUwNTYwNiIM5CJHSuHnYhjVFNBVKooCvmokdCGCJJHcy5q7TYt8OPv%2FHD642SvzHSemeW7hV2%2F643sJniww4EcWAGInH9Udtxm3q9Ch7J8aEjvqL7jcFXNLePFoRaOa5nh6AeCo2tSQhPgXsKsvnAJesXVH8TmxQlv41YOo7%2BO%2BEgWTDbrCuBIgDxiCmCQHXluPy9MD2w9k3wTLlY%2FYPOXcuttVf88kO83zZ3alQD63jmlDMdbe63eW0n0bZwVUWiNwEr12N%2Bp6PoDWs1%2F%2F5k%2Bmt7O2LYCAjQxIYjggh8iIN4QRhjfA6oBZGQD99Xy%2FhmOFXyvtFLQDC%2BSXmvU9BNoFwLZClmMEqxyUOdXxvZbQT8QFdhxdzRZwWRMo%2FUysg90w7YbziAY6mgHOYRAbpM3vPjXcqDPi9%2BsQaP6YUm%2FOeanciG6hj3PahNL8cbML48MVAdr2jaVEeX4s9IffgY55jIxrz9vLqudBWrFUaVAntOh49j%2FOaPr%2Fwss8etDrekssJqp3MBp7VAJ01UKheq3dGbgNlbevMxuHpU9sIZCvaqHV0yQ06YBIZoHOpCowZaSmCL30Fo4hHYMjlxf5wJjULxxd",
+            "validFor": "30 Minutes"
+        }
+    ]
+}
+|4xx              |
+
+
+#### 4.3 Send Acknowledgement
+Use the acknowledgment API to notify CFT about the files downloaded. This is optional.
+
+![Display Step5](./images/Sandbox_acktran
+'.jpg)
+
+##### 4.3.1 Resource Information
+|Method      |PUT
+|------------|-------------------------
+|URL         |https://api-sandbox.gdscft.govtechstack.sg/sandbox/v1/cft/transactions/{transactionid}/ack
+|What it does|Notifies Sender about the files downloaded in a transaction
+
+##### 4.3.2 Request
+###### 4.3.2.1 Headers
+Refer 2.2.1 \(of Sender\)
+
+##### 4.3.3 Response
+###### 4.3.3.1 Sample Response
+|Code             |Response
+|-----------------|-----------------------------
+|200              |
+|4xx              |
+
+
+## Support
+Support is delivered over telegram channel and during office hours.
```

Comment:
Step 8 is the last step, maybe something to signal that the reader has reached the end of the doc will be gd